helpers/validator.js

```diff
@@ -168,8 +168,8 @@ const getPendingClaimedAccounts = callback => {
 };
 
 const isEmail = email => {
-    const reg = /^\w+@[a-zA-Z0-9]{2,10}(?:\.[a-z]{2,4}){1,3}$/;
-    return reg.test(email);
+    const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
this is too long, not readable

Reply:
/^\w+@[a-zA-Z0-9]{2,30}(?:\.[a-z]{2,30}){1,3}$/

Reply:
a valide email is like  ：
 xxx@xx.com  start with a character so, we put do 
reg=/^\w+([0-9\.]{0,20})+[a-zA-Z0-9]{0,20}@[a-zA-Z0-9]{2,20}(?:\.[a-z]{2,20}){1,3}$/